app.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const app = express();
+const { getApi } = require('./controller');
+
+// app.use(express.json());
```

Comment:
Please remove this until you need it, it's best to avoid pushing commented out code.